usr.bin/paste/paste.c

```diff
@@ -44,11 +47,20 @@
 #include <unistd.h>
 #include <wchar.h>
 
+typedef struct _list {
```

Comment:
I'd recommend, if possible, move non-functional changes ( refactors ) to separated commit. This will make reviewing much easier.

Reply:
Almost every change herein was necessary to smoothly introduce Capsicum and enter capability mode, so the refactoring is closely tied to this effort, as it helped streamline file management and error handling.

Reply:
Alright, that makes sense. I’ll take care of it shortly, roger that.